server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -372,7 +389,7 @@ export class TenantManager {
 
 			const encryptedTenantKey2 = tenantDocument.secondaryKey;
 			const tenantKey2 = encryptedTenantKey2
-				? this.secretManager.decryptSecret(encryptedTenantKey2)
+				? this.secretManager.decryptSecret(encryptedTenantKey2, encryptionKeyVersion)
```

Comment:
Just to confirm, this change will be deployed only after we have updated the existing tenantKeys in db to be encrypted by the new 2023 key. Is that correct?

Reply:
No. The code change is deployed before we run the script. encryptionKeyVersion theoretically could either be empty or an enum from `EncryptionKeyVersion`. After we run the script, encryptionKeyVersion will be changed from empty to "2023".